public/assets/main.css

```diff
@@ -11,11 +11,24 @@ a {
   text-underline-offset: 0.2em;
 }
 
-button {
-  margin: 0.5rem 0.5rem 0 0;
+button, .button-link {
+  font-size: 14px;
+  background: whitesmoke;
+  color: black;
+  border: solid 1px black;
+  border-radius: 5px;
+  padding: 0.2rem 1rem;
+  margin: 0.5rem 1rem 0 0;
+  line-height: 1rem;
+  cursor: pointer;
+  text-decoration: none;
 }
-button + button {
-  margin-left: 0.5rem;
+button.small {
+  padding: 0.2rem 0.5rem;
+  margin-right: 0.5rem;
+}
+button:last-child, button.par_price:nth-of-type(6n), button.sell_share:nth-of-type(3n) {
```

Comment:
what does 6n and 3n mean

Reply:
x*n-th element (6th, 12th etc.)
Used here to remove margin-right _only_ from the last element per row of (possibly) multiple rows [1846!] so the full width can be occupied.